analysis/trigger_performance/PHL_trigger_report.Rmd

```diff
@@ -0,0 +1,349 @@
+---
```

Comment:
I'd recommend pushing the HTML on GH in this case. It makes it possible to link to a readable and user-friendly version of the report in emails, reports, and presentations. It'd also give us a chance to review the report in its readable/visual format.